FILE packages/server/src/utils/project-icon.ts
RANGE {"path":"packages/server/src/utils/project-icon.ts","old_lines":[10,13],"new_lines":[10,16],"diff_block_lines":[5,12],"hunk_header":"@@ -10,4 +10,7 @@ export const ICON_PATTERNS = [","substantive_changed_lines":3,"comparison":"parent"}
@@ -10,4 +10,7 @@ export const ICON_PATTERNS = [
   "favicon.png",
   "favicon.svg",
+  "favico.ico",
+  "favico.png",
+  "favico.svg",
   "icon.png",
   "icon.svg",
